src/commands/sns/make_upgrade_canister_proposal.rs

```diff
@@ -62,6 +63,10 @@ pub struct MakeUpgradeCanisterProposalOpts {
     /// Path to the binary file containing argument to post-upgrade method of the new canister WASM.
     #[arg(long, conflicts_with = "canister_upgrade_arg")]
     canister_upgrade_arg_path: Option<String>,
+
+    /// The install mode.
+    #[arg(long, short, value_parser = ["install", "reinstall", "upgrade"])]
```

Comment:
Quill tries to be very conservative about short flags. I don't think we have any besides -y and neuron-manage's -a.

```suggestion
    #[arg(long, value_parser = ["install", "reinstall", "upgrade"])]
```